usaspending_api/references/management/commands/loadtas.py

```diff
@@ -22,9 +21,8 @@ def add_arguments(self, parser):
     def handle(self, *args, **options):
         is_remote_file = len(options['location'][0].split('.')) == 1
         if is_remote_file:
-            s3connection = boto.s3.connect_to_region(os.environ.get('USASPENDING_AWS_REGION'))
-            s3bucket = s3connection.lookup(options['location'][0])
-            file_path = s3bucket.get_key('cars_tas.csv')
+            s3connection = boto3.client('s3', region_name=os.environ.get('USASPENDING_AWS_REGION'))
```

Comment:
can you replace all AWS region env vars to instead use settings.py's `USASPENDING_AWS_REGION`

Reply:
I could, are we sure that setting is populated on all the instances? I kept them the same to not run into any issues with operations.